solana/pyth2wormhole/program/src/attest.rs

```diff
@@ -248,15 +262,30 @@ pub fn attest(ctx: &ExecutionContext, accs: &mut Attest, data: AttestData) -> So
     );
     solana_program::program::invoke(&transfer_ix, ctx.accounts)?;
 
+    let payload = batch_attestation.serialize().map_err(|e| {
+        trace!(&e.to_string());
+        ProgramError::InvalidAccountData
+    })?;
+
+    // Adjust message account size if necessary
+    if !accs.wh_message.is_initialized() && accs.wh_message.payload.len() != payload.len() {
```

Comment:
Curious - why do we only resize if the account is un-initialized? Does the account ever get reset back to un-initialized, or do we only ever resize once?